styles/utils.css

```diff
@@ -5,7 +5,7 @@
     --white: #ffffff;
 	--offwhite: #f7f7f7;
 	--black: #282828;
-	--gray: #9a9a9a;
+	--grey: #9a9a9a;
     --purple: purple;
     --info: skyblue;
     --red: red;
```

Comment:
same as above, don't use variable name as color name. We use variables we can change the color easily at one place and it will get changed everywhere automatically. But if you name it white, you can't change the color as name wont make sense

Reply:
Got it, thanks